Dockerfile

```diff
@@ -15,9 +15,17 @@ RUN if [ "$BUILDPLATFORM" = "$TARGETPLATFORM" ]; then go test -v -race github.co
 
 RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/ratelimit -ldflags="-w -s" -v github.com/replicon/ratelimit/src/service_cmd && \
  CGO_ENABLED=0 GOOS=linux go build -o /go/bin/ratelimit_config_check -ldflags="-w -s" -v github.com/replicon/ratelimit/src/config_check_cmd
-FROM alpine:3.11 AS final
-RUN apk --no-cache add ca-certificates
+
+
+FROM python:3.10-alpine AS final
```

Comment:
rollback to a previous alpine release?

Reply:
that's actually python 3.10 on "latest" alpine...

Reply:
aww awesome

Reply:
FYI updated to pin to alpine 3.16